fe2-android/app/src/test/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionSetupTest.java

```diff
@@ -33,29 +36,13 @@ public class ElectionSetupTest {
   private final List<String> question = Arrays.asList("which is the best ?", "who is best ?");
   private final String laoId = "my lao id";
   private final ElectionSetup openBallotSetup =
-      new ElectionSetup(
-              electionVersion,
-          electionSetupName,
-          creation,
-          start,
-          end,
-          votingMethod,
-          writeIn,
-          ballotOptions,
-          question,
-          laoId);
+          new ElectionSetup(
+                  writeIn, electionSetupName, creation, start, end, votingMethod, laoId, ballotOptions, question, electionVersion
```

Comment:
The point was to use ElectionVersion.OPEN_BALLOT directly on this line, instead of defining a variable.